src/standard-live-connect.js

```diff
@@ -103,17 +104,19 @@ function _processArgs (args, pixelClient, enrichedState) {
  * @return {StandardLiveConnect|null}
  * @private
  */
-function _getInitializedLiveConnect (liveConnectConfig) {
+function _getInitializedLiveConnect (liveConnectConfig, initializationCallback) {
   try {
     if (window && window.liQ && window.liQ.ready) {
-      const mismatchedConfig = window.liQ.config && _configMatcher(window.liQ.config, liveConnectConfig)
-      if (mismatchedConfig) {
-        const error = new Error()
-        error.name = 'ConfigSent'
-        error.message = 'Additional configuration received'
-        emitter.error('LCDuplication', JSON.stringify(mismatchedConfig), error)
+      if (qualifiedConfig(window.liQ.config) && !initializationCallback) {
```

Comment:
Does this mean we will emit an error like we used to but only for the clients that use live-connect module
and this won't be the same for the builder?

Reply:
I adjusted the condition a little. The idea to prioritize the config with an appId that comes from the builder higher than the current config because the builder will also initialize the privacy properties and the SyncContainer.